<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPanel</class>
 <widget class="QWidget" name="MainPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>16</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>151</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="WifiLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icons/wifi-100.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="BatteryLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icons/bat-100-charging.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ClockLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>1:14 AM</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
